example_config.yaml

```diff
@@ -1,49 +1,89 @@
 chihaya:
+  # The interval communicated with BitTorrent clients informing them how
+  # frequently they should announce in between client events.
   announce_interval: 15m
-  prometheus_addr: localhost:6880
+
+  # The network interface that will bind to an HTTP endpoint that can be
+  # scraped by an instance of the Prometheus time series database.
+  # For more info see: https://prometheus.io
+  prometheus_addr: 0.0.0.0:6880
+
+  # The maximum number of peers returned in an announce.
   max_numwant: 50
+
+  # The default number of peers returned in an announce.
   default_numwant: 25
 
+  # This block defines configuration for the tracker's HTTP interface.
+  # If you do not wish to run this, delete this section.
   http:
+    # The network interface that will bind to an HTTP server for serving
+    # BitTorrent traffic.
     addr: 0.0.0.0:6881
+
+    # When enabled, the IP address used to connect to the tracker overrides
+    # the value clients advertise.
     allow_ip_spoofing: false
+
+    # The HTTP Header containing the IP address of the client.
+    # This is only necessary if using a reverse proxy.
```

Comment:
IIRC, if the client doesn't provide an IP we fallback to the RemoteAddr as a last resort.